docs/dashboards/data-quality-dashboard.asciidoc

```diff
@@ -0,0 +1,59 @@
+[[data-quality-dash]]
+= Data Quality dashboard
+
+The Data Quality dashboard shows you whether your data is effectively mapped to the https://www.elastic.co/guide/en/ecs/current/ecs-reference.html[Elastic Common Schema] (ECS). Successful {ref}/mapping.html[mapping] enables you to search, visualize, and interact with your data throughout {es-sec} and {kib}.
+
+image::images/data-qual-dash.png[The Data Quality dashboard]
+
+You can use the Data Quality dashboard to:
+
+* Check one or multiple indices (by default, the indices used by {es-sec}) for unsuccessful mappings, to help you identify problems.
+* View detailed information about the fields in checked indices.
+* Track unsuccessful mappings by creating a case or markdown report based on Data Quality results.
```

Comment:
```suggestion
* Track unsuccessful mappings by creating a case or Markdown report based on data quality results.
```

Reply:
I'm not sure if you're referring to the dashboard results here, so feel free to reject this change if you're using the short-hand version of the dashboard name.